api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -430,7 +430,7 @@ def test_multiplexed_metadata(self):
             "disease_ontology_term_id",
             "droplet_filtering_method",
             "filtered_cell_count",  # with non-multiplexed
-            "filtered_cells",
+            # "filtered_cells",
```

Comment:
Is there a reason that we keep this `filtered_cells` property around when testing Multiplexed? In `metadata_file.py`, its one of the properties that we transform, replacing it with `filtered_cell_count`. With Single-cell we've removed it.

Reply:
This is ok to remove here and not comment out